packages/sil-voting-app/README.md

```diff
@@ -0,0 +1,44 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
Update this file with a basic message. Something like "This is the decentralized IOV's voting app for autonomous organisations" or something like that.